tests/e2e/test_wormhole.js

```diff
@@ -0,0 +1,77 @@
+const path				= require('path');
+const log				= require('@whi/stdlog')(path.basename( __filename ), {
+    level: process.env.LOG_LEVEL || 'fatal',
+});
+
+// const why				= require('why-is-node-running');
+const expect				= require('chai').expect;
+const { structs, ...lair }		= require('@holochain/lair-client');
+
+
+const { init }				= require("../../build/wormhole.js");
+
+const LAIR_SOCKET			= path.resolve( __dirname, '../lair/socket' );
+const CONDUCTOR_SOCKET			= path.resolve( __dirname, '../shim_socket' );
+
+
+describe("Wormhole tests", () => {
+    let wormhole;
+    let fake_signature			= Buffer.from("ea067251189fa64a65a33548dc8c4e2989b50d27ec915391bc1491bd52047621d27b097aa352d5470baa9356260cda206d77da5c13d32ab8465f2265bccd7970", "hex");
+
+    before(async () => {
+	wormhole			= await init( LAIR_SOCKET, CONDUCTOR_SOCKET, async function (pubkey, message) {
+	    return fake_signature;
+	});
+    });
+    after(async () => {
+	await wormhole.stop();
+    });
+
+    it("should process request and respond", async () => {
+	let shim_client, resp;
+	let recv_unlock			= false;
+
+	try {
+	    shim_client			= await lair.connect( CONDUCTOR_SOCKET );
+	    log.info("Lair client", shim_client );
+
+	    shim_client.on('UnlockPassphrase', request => {
+		log.normal("Received UnlockPassphrase request");
+		recv_unlock		= true;
+		request.reply( "Passw0rd!" );
+	    });
+
+	    resp			= await shim_client.request( new structs.TLS.CreateCert.Request( 512 ) );
+	} catch (err) {
+	    console.error( err );
+	} finally {
+	    shim_client.destroy();
+	}
+
+	expect( resp.get(0)		).to.be.a('number');
+	expect( resp.get(1)		).to.be.a('uint8array');
+	expect( resp.get(2)		).to.be.a('uint8array');
+	expect( recv_unlock		).to.be.true;
+    });
+
+    it("should process request and respond", async () => {
```

Comment:
This test and the previous one have the same name. Can you give them different names?